lib/assessment.js

```diff
@@ -4,44 +4,111 @@
 // Write a function that takes zero or more arguments
 // and returns the minimum from that set
 // assume that all the argument are numbers
-var min = function min() {
+// var min = function min([1, 3, 4, 2]) {
+
+//   var findMin = 0;
+
+//   for(var i = 0; i < min.length; i++) {
+//     return Math.min.apply();
+//   };
+
+// };
+
+// I am not sure how to write this correctly, but I would use Math.min to find the minimum number in an array.
+
+
+// correct answer:
+var min = function(){
+  var result = arg [0]; // first spot in the array - this is where to store the result
+  for(var index; index < arguments.length; index++); // iterate over the array
+    if(arguments[index] < result);
+      result = arguments[index];
+}
+return result; // have to see everything in the result, so return has to be outside the function
+
+
 
-};
 
 // Question 2
 // Write a function that takes zero or more arguments
 // and returns the sum of that set
 // assume that all the argument are numbers
-var sum = function sum() {
+// var sum = function sum([1, 2, 3, 4]) {
+
+//   var sumTotal = 0;
+//   for(var i = 0; i < sum.length; i++); {
+//     sumTotal += sum[i];
+//   }
+
+//   return sumTotal;
+
+// };
+
+// correct answer:
+var sum = function sum(){
+  var result = 0;
+  for (var index = 0; index < args.length; index ++);
+    result += arg[index];
+}
+return result;
```

Comment:
This is another `return` from outside a function. It will do nothing but throw a syntax error.